src/packaging.rs

```diff
@@ -227,13 +225,30 @@ pub fn package_conda(
 
     // print sorted files
     tracing::info!("\nFiles in package:\n");
-    tmp.files
+    let mut files = tmp
+        .files
         .iter()
         .map(|x| x.strip_prefix(tmp.temp_dir.path()))
-        .collect::<Result<Vec<_>, _>>()?
-        .iter()
-        .sorted()
-        .for_each(|f| tracing::info!("  - {}", f.to_string_lossy()));
+        .collect::<Result<Vec<_>, _>>()?;
+    files.sort_by(|a, b| {
+        let a_is_info = a.components().next() == Some(Component::Normal("info".as_ref()));
+        let b_is_info = b.components().next() == Some(Component::Normal("info".as_ref()));
+        match (a_is_info, b_is_info) {
+            (true, true) | (false, false) => a.cmp(b),
+            (true, false) => std::cmp::Ordering::Greater,
+            (false, true) => std::cmp::Ordering::Less,
+        }
+    });
+    files.iter().for_each(|f| {
+        if f.components().next() == Some(Component::Normal("info".as_ref())) {
+            tracing::info!(
+                "  - {}",
+                console::style(f.to_string_lossy()).black().bright()
```

Comment:
I wonder if `.dim()` is better?

Reply:
yup, b19ca0ebc71ab726c9758f56906018beaed2d3c9